test/extended/util/docker.go

```diff
@@ -0,0 +1,60 @@
+package util
+
+import (
+	dockerClient "github.com/fsouza/go-dockerclient"
+	tutil "github.com/openshift/origin/test/util"
+)
+
+func CorruptImage(corruptee, corruptor string) error {
+	client, dcerr := tutil.NewDockerClient()
+	if dcerr != nil {
+		return dcerr
+	}
+	opts := dockerClient.TagImageOptions{
+		Repo:  corruptee,
+		Tag:   "latest",
+		Force: true,
+	}
+	terr := client.TagImage(corruptor, opts)
+	return terr
+}
+
+func ResetImage(name string) error {
+	client, dcerr := tutil.NewDockerClient()
+	if dcerr != nil {
+		return dcerr
+	}
+	opts := dockerClient.PullImageOptions{
+		Repository: name,
+		Tag:        "latest",
+	}
+	perr := client.PullImage(opts, dockerClient.AuthConfiguration{})
+	return perr
+}
+
+func CompareImages(comp1, comp2 string, returnID1, returnID2 *string) error {
```

Comment:
go supports multiple return values.

also i'd rename this to something like "GetImageIDForTag" since that's really what you're doing, not comparing them.

whether you want to keep it returning 2 values (for optimization) vs just returning a single value and calling it twice (which makes more generic sense) i guess i don't have a strong opinion.  Though a third option would be to have it take an array of tags to look up and return an array of ids, with nil values in entries where no id was found.  (and probably return an Error in that case as well)